test/tools/libtesteth/TestSuite.h

```diff
@@ -42,12 +42,15 @@ class TestSuite
 	// Main test executive function. should be declared for each test suite. it fills and runs the test .json file
 	virtual json_spirit::mValue doTests(json_spirit::mValue const&, bool) const = 0;
 
-	// Execute all tests from _folder
+	// Execute all tests from suiteFolder()/_testFolder/*
+	// This functions checks that tests in the repo are updaed with /src/suiteFillerFolder()/*Filler tests
+	// Then it looks for a Filler.json file in /src/suiteFillerFolder()/_testFolder and generates the actual tests
+	// If src test file is named Copier it will just copy it from /src folder because such test is crafted and
+	// it's RLP could not be generated by the client
 	void runAllTestsInFolder(std::string const& _testFolder) const;
 
-	// Copy .json tests from the src folder to the dest folder because such test is crafted manually and could not be filled.
-	// Used in bcForgedTest, ttWrongRLP tests and such.
-	void copyAllTestsFromFolder(std::string const& _testFolder) const;
+	//Execute Filler.json or Copier.json test file in a given folder
```

Comment:
Document what happens when the `_jsonFileName` is neither a `Copier.json` or a `Filler.json`.